requirements.txt

```diff
@@ -22,7 +22,7 @@ sympy >= 1.12
 
 # packages that can't be installed with conda but can be installed with pip on solo #pippkgs
 ipykernel >= 6.29.3
-opencv-contrib-python == 4.5.5.64
+opencv-contrib-python >= 4.8.1.78
```

Comment:
OpenCV versions >= 4.10.0.84 are incompatible with numpy 1.x. We need to do one of the following:

Option A: "numpy >= 2.1.2", "opencv-contrib-python >= 4.10.0.84"
Option B: "numpy <= 2", "opencv-contrib-python >= 4.8.1.78 < 4.10.0.84"

My vote is to try option A and see if all the tests pass. Keep in mind that the migration from numpy 1 to numpy 2 _could potentially break some things_. If we need to do any changes we can reference [https://numpy.org/devdocs/numpy_2_0_migration_guide.html](https://numpy.org/devdocs/numpy_2_0_migration_guide.html).

Reply:
We will also need to replace all imports matching "from cv2 import cv2" to just be "import cv2".